chaco/tools/tests/test_image_inspector.py

```diff
@@ -83,7 +83,7 @@ class TestImageInspectorToolGray(BaseImageInspectorTool, TestCase):
     def setUp(self):
         values = np.arange(4).reshape(2, 2)
         self.plot = create_image_plot(values)
-        super(TestImageInspectorToolGray, self).setUp()
+        BaseImageInspectorTool.setUp(self)
```

Comment:
Why exactly can't this be `super().setUp()`?  I notice the tests all still pass if that change is made.

Reply:
we discussed this in person - this is part convention and part best practice afaik - especially because we have a few places where we need to call `setUp` on more than one baseclass.